pkg/scheduler/metrics/metrics.go

```diff
@@ -128,11 +129,12 @@ var (
 	)
 	SchedulingAlgorithmPredicateEvaluationDuration = metrics.NewHistogram(
```

Comment:
`SchedulingAlgorithmPredicateEvaluationDuration` -> `DeprecatedSchedulingAlgorithmPredicateEvaluationDuration`

Same for other deprecated metrics.